src/test/java/hudson/plugins/logparser/LogParserTestCase.java

```diff
@@ -107,7 +108,7 @@ protected FreeStyleProject setupFreeStyleProject(boolean broken) throws URISynta
         if (Functions.isWindows()) {
             project.getBuildersList().add(new BatchFile("@echo \"This is a LogParserTest\" && echo \"INFO\" && echo \"WARN\" && echo \"ERROR\""));
         } else {
-            project.getBuildersList().add(new Shell("echo \"This is a LogParserTest\" && echo \"INFO\" && echo \"WARN\" && echo \"ERROR\""));
+            project.getBuildersList().add(new Shell("#!/bin/bash -l\necho \"This is a LogParserTest\" && echo \"INFO\" && echo \"WARN\" && echo \"ERROR\""));
```

Comment:
This is a workaround for the `-x` that the `Shell` builder sends to the shell [JENKINS-14731]. For Windows this is already handles by the `@`.